pkg/compute/build.go

```diff
@@ -129,17 +123,166 @@ func (c *BuildCommand) Exec(in io.Reader, out io.Writer) (err error) {
 	progress.Step("Creating package archive...")
 
 	dest := filepath.Join("pkg", fmt.Sprintf("%s.tar.gz", name))
-	err = createPackageArchive([]string{
+
+	files := []string{
 		ManifestFilename,
 		"Cargo.toml",
-		"bin",
-		"src",
-	}, dest)
+	}
+
+	ignoreFiles, err := getIgnoredFiles(IgnoreFilePath)
+	if err != nil {
+		return err
+	}
+
+	binFiles, err := getNonIgnoredFiles("bin", ignoreFiles)
 	if err != nil {
 		return err
 	}
+	files = append(files, binFiles...)
+
+	if c.includeSrc {
+		srcFiles, err := getNonIgnoredFiles("src", ignoreFiles)
```

Comment:
If we decide to change the flag to be less rust-specific, we'll want to map what it means per supported language in the future, but this implementation can stay for now.

Reply:
Have added TODO comment :+1: